content/pairing.md

```diff
@@ -0,0 +1,248 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.6.0
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Pairing Jupyter notebooks and MyST-NB
+
+## What you'll do
+This guide will keep a Jupyter notebook synced _or paired_ between
+`.ipynb` and `.md`. 
+
+## What you'll learn
+- The difference between Jupyter's json format and MyST-NB's markdown
+  format
+- The benefits and drawbacks of json and markdown
+- How to keep `.ipynb` and `.md` files in sync
+
+## What you'll need
+- [Jupyter](https://jupyter.org/)
+- [Jupytext](https://jupytext.readthedocs.io/en/latest/index.html)
+
+---
+## Background
+
+The [NumPy tutorials](https://github.com/numpy/numpy-tutorials) are
+reviewed and executed as [MyST-NB](https://myst-nb.readthedocs.io/)
+notebooks.  Content is easier to review in this markdown format. You can
+keep your `.ipynb` in sync with the content on NumPy tutorials. The
+NumPy tutorials use
+[Jupytext](https://jupytext.readthedocs.io/en/latest/index.html) to
+convert your `.ipynb` file to [MyST
+Markdown](https://github.com/mwouts/jupytext/blob/master/docs/formats.md#myst-markdown)
+format.
+
+Jupyter notebooks are stored on your disk in a
+[json](https://www.json.org/json-en.html) format. The json format is
+very powerful and allows you to store almost any input and output that
+Python libraries can create. The drawback is that json is difficult to
+read and compare files. 
```

Comment:
This may be a nit, but it is not necessarily hard to compare json files. The difficulty here is seeing the final rendered notebook for the json file, which is a problem specific to notebooks and not inherent to json files. You may want to reword this, but here's a suggestion:

```suggestion
Python libraries can create. The drawback is that it is hard to see and compare changes made in the notebook file when reviewing pull requests, because this means the reviewers are looking only at the raw json files. 
```

Reply:
I think that's a nice distinction to make, thanks @melissawm.